storage/storage.go

```diff
@@ -1203,6 +1208,14 @@ type ObjectAttrs struct {
 	// Etag is the HTTP/1.1 Entity tag for the object.
 	// This field is read-only.
 	Etag string
+
+	// A user-specified timestamp which can be applied to an object. This is
+	// typically set in order to use the CustomTimeBefore and DaysSinceCustomTime
+	// LifecycleConditions to manage object lifecycles.
+	//
+	// CustomTime cannot be removed once set on an object, and if its value is
+	// already set, it cannot be changed to an earlier value.
```

Comment:
CustomTime can be moved forward into the future. I think it's there but would help to call it out explicitly.

Reply:
Clarified this.